apps/election-manager/src/contexts/AppContext.ts

```diff
@@ -36,7 +36,7 @@ export interface AppContextInterface {
   saveIsOfficialResults: () => Promise<void>;
   resetFiles: (fileType: ResultsFileType) => Promise<void>;
   usbDriveStatus: usbstick.UsbDriveStatus;
-  usbDriveEject: () => Promise<void>;
+  usbDriveEject: (currentUser: string) => Promise<void>;
```

Comment:
This is `string` in a bunch of places, but it should be `LoggingUserRole`, right?